### DISCUSSION TITLE
SCAM

### DISCUSSION THREAD

**Discussion by h3xd4ck** [community/community#106928] [Repositories] [answered]
### Select Topic Area

Show & Tell

### Body

https://github.com/UniRevendBypass/Universal_Crypter/tree/9c615d274e46fc5a002a8b0c2892c03f6b4c1c02
if u wanna buy something from this guy u will be scammed, be carefull

**Comment by queenofcorgis** [comment, marked as answer]
Hi @h3xd4ck 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
